core/src/commonMain/kotlin/com/sourcepoint/mobile_core/models/consents/USNatConsent.kt

```diff
@@ -23,8 +24,8 @@ data class USNatConsent(
 
     @Serializable
     data class USNatUserConsents (
-        val vendors: List<USNatConsentable> = emptyList(),
-        val categories: List<USNatConsentable> = emptyList()
+        var vendors: List<USNatConsentable> = emptyList(),
+        var categories: List<USNatConsentable> = emptyList()
     )
 
```

Comment:
@Nevazhnovu rather than making the attributes mutable (changing from `val` to `var`) we can use the `.copy` function whenever we need to update attribute values. 

Let's say you want to update `uuid` and the `applies` fields, but keep everything else the same:
```kotlin
state.usnatConsent = state.usnatConsent.copy(
   uuid = "foo",
   applies = false
)
```

Keeping things immutable are, as a rule of thumb, less prone to bugs, because there's less changes and side effects going on.

The same apply to the other classes.